src/options/options.html

```diff
@@ -79,17 +85,59 @@
             </p>
           </div>
         </p>
+        </div>
+        <script>
```

Comment:
This script should go in the `options.js` file, in the `addListeners` function. It doesn't even work for me otherwise: `Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”).`é

Reply:
Done.